packages/nvidia_nat_core/tests/nat/eval/test_evaluate.py

```
@@ -316,6 +316,65 @@ async def mock_error_session(user_id=None):
+    ])
+    evaluation_run.eval_config.general.max_concurrency = 1
+
+    mock_runner = AsyncMock()
+
+    call_count = 0
+
+    async def mock_result():
+        nonlocal call_count
+        call_count += 1
+        if call_count == 2:
+            raise RuntimeError("Simulated workflow failure")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)